src/openrct2/ride/TrackData.cpp

```diff
@@ -5956,7 +5956,7 @@ static constexpr uint8_t TrackSequenceElementAllowedWallEdges[][MaxSequencesPerP
 static_assert(std::size(TrackSequenceElementAllowedWallEdges) == TrackElemType::Count);
 
 /** rct2: 0x0099423C */
-static constexpr uint16_t TrackFlags[] = {
```

Comment:
Is this change needed? Weere we at the verge of overflow?

Reply:
Yes, `TRACK_ELEM_FLAG_CAN_BE_PARTLY_UNDERGROUND` did not fit into a uint16_t, so I had to enlarge it.